src/components/shared/__tests__/AddRemoveForm.spec.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import AddRemoveForm from '../AddRemoveForm'
+import { shallow } from 'enzyme'
+import mockData from 'mockData'
+
+describe('<AddRemoveForm />', () => {
+  let wrapper
+  const mockProps = {
+    terms: {
+      title: 'Mock Terms',
+      // labelProp: 'name', // default
+      fetchItems: () => mockData.terms.getAll()
+    },
+    questions: {
+      title: 'Mock Questions',
+      labelProp: 'title',
+      fetchItems: () => mockData.questions.getAll()
+    }
+  }
+
+  beforeEach(() => {
+    wrapper = shallow(<AddRemoveForm {...mockProps.terms} />)
+  })
+
+  it('should render props.title', () => {
+    const result = wrapper.find({ 'data-test': 'title' }).text()
+    expect(result).toBe(mockProps.terms.title)
+  })
+
+  it('should call props.fetchItems() and save result', () => {
+    const results = wrapper.state('items')
+    expect(mockData.terms.getAll).toHaveBeenCalledTimes(1)
+    expect(results).toBe(mockData.terms.items)
+  })
```

Comment:
I've never checked state with enzyme before. Usually this data would be stored in redux, but this was nice and clean to test!